initfiles/componentfiles/thor/run_thor

```diff
@@ -43,6 +42,13 @@ while [ 1 ]; do
     fi
     $deploydir/makethorgroup
     sort $instancedir/slaves | uniq > $instancedir/uslaves.start
+    diff $instancedir/uslaves.start <(${INSTALL_DIR}/sbin/configgen -env ${CONFIG_DIR}/${ENV_XML_FILE} -c ${THORNAME} -listall2 | awk 'BEGIN {FS=",";} /ThorSlaveProcess/ { print $3 }' | sort | uniq) &>/dev/null
```

Comment:
a -t thor would do no harm, but unfortunately you'll still need the filter as there is no 'slave' type.